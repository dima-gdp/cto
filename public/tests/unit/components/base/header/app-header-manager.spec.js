import { mount } from '@vue/test-utils'
import AppHeaderManager from '@/components/base/header/app-header-manager.vue'
import Vue from 'vue'
import Vuex from 'vuex'

Vue.use(Vuex)

describe('@/components/base/header/app-header-manager.vue', () => {
  let wrapper
  let store

  const createComponent = () => {
    store = new Vuex.Store({
      state: {
        event: {
          data: {
            email: 'manager@pointreg.com',
            phone: {
              info: 'Елизавета Гаврилова - Менеджер POINTREG',
              number: '+7 (920) 258-13-50',
              extension: '256',
            },
          },
        },
      },
    })

    wrapper = mount(AppHeaderManager, {
      store,
    })
  }

  afterEach(() => {
    wrapper.destroy()
  })

  it('Когда есть все данные для отрисовки(телефон, телефон добавочный, почта, инфо)', () => {
    createComponent()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
