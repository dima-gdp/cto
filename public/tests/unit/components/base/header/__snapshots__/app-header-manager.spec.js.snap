// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@/components/base/header/app-header-manager.vue Когда есть все данные для отрисовки(телефон, телефон добавочный, почта, инфо) 1`] = `
<div class="info-manager">
  <p class="info-manager__title">Елизавета Гаврилова - Менеджер POINTREG</p> <a href="tel:+7 (920) 258-13-50" class="info-manager__link info-manager__link--nw">
    +7 (920) 258-13-50
    <span>256</span></a> <a href="mailto:manager@pointreg.com" class="info-manager__link">manager@pointreg.com</a>
</div>
`;
